backend/server/src/models.rs

```diff
@@ -175,10 +175,36 @@ pub trait Aggregate<T: Queryable<(BigInt, Text), Pg>> {
         Self::field()
     }
 
+    fn recent_get(
+        language: &str,
+        conn: &Pool<ConnectionManager<PgConnection>>,
+        options: &HashMap<String, String>,
+        limit: Option<i64>,
+        interval: Option<&str>
+    ) -> Result<Vec<T>> {
+        let connection = conn.get()?;
+        let mut interval_str = String::from("created_at > NOW() - interval '");
```

Comment:
This is ripe for sql injection if the user controls the interval. Diesel’s type system should be able to handle this, no?